recipes/hictk/build.sh

```diff
@@ -2,10 +2,10 @@
 
 export CONAN_NON_INTERACTIVE=1
 
-export CMAKE_BUILD_PARALLEL_LEVEL=${CPU_COUNT}
-export CTEST_PARALLEL_LEVEL=${CPU_COUNT}
+export CMAKE_BUILD_PARALLEL_LEVEL=$CPU_COUNT
```

Comment:
Why ?
This reverts someone's work on applying best practices.

Reply:
Thanks for your comment.
I believe I am the only one who has contributed code to this recipe so far :)

I am aware of the cases where curly braces are required to reference variables (and I make sure I use them in those cases), but I don't think that "always use curly braces" is a widely accepted best practice.

I personally find that using them everywhere makes the code a bit more difficult to read, but if "always use curly braces" is the bioconda way, I will revert that change.